### ISSUE TITLE
Copy track icon into language track repository

### ISSUE THREAD

**Issue by kytrinyx** [exercism/xpython#338]
Right now all of the icons used for the language tracks (which can be seen at http://exercism.io/languages) are stored in the exercism/exercism.io repository in `public/img/tracks/`. It would make a lot more sense to keep these images along with all of the other language-specific stuff in each individual language track repository.

There's a pull request that is adding support for serving up the track icon from the x-api, which deals with language-specific stuff.

In order to support this change, each track will need to

- create a directory named `img/` at the root of this repository if it doesn't already exist, then
- find the correct icon in https://github.com/exercism/exercism.io/tree/master/public/img/tracks, and
- stick it in the `img/` directory, and importantly
- **name the image `icon.png`**

In other words, at the end of it you should have the following file:

```
./img/icon.png
```

See https://github.com/exercism/exercism.io/issues/2925 for more details.

**Comment by bordeltabernacle** [comment]
In case it hasn't been noticed, this issue can be closed after PR #339 @kytrinyx @Dog :+1:

**Comment by Dog** [comment]
Thanks @bordeltabernacle
